uint16[] id_list
float32[] input_torque_list
float32[] torque_ramp_rate_list
